src/test/Header.test.js

```diff
@@ -3,7 +3,7 @@ import { render } from '@testing-library/react';
 import { MemoryRouter as Router } from 'react-router-dom';
 import Header from '../components/Header/Header';
 
-it('It should render the component Correctly', () => {
+it('Header component is rendered correctly', () => {
   const header = render(
     <Router>
       <Header />
```

Comment:
- [ ] you should do the same for the details page so please make sure the details should look like the the design provided in font space and layouts for more info look in above comments 
- Yours
<img width="506" alt="Screenshot 2022-12-10 at 20 24 32" src="https://user-images.githubusercontent.com/61790587/206872275-04607179-5832-4021-b176-0ebc2074fd41.png">
- Design 
<img width="348" alt="Screenshot 2022-12-10 at 20 26 17" src="https://user-images.githubusercontent.com/61790587/206872284-f18f651c-3e21-432a-9d24-f15e9a6631d3.png">